src/lib/network/clients/save_network.rb

```diff
@@ -298,12 +291,32 @@ def configure_lan
       # skipped or even only done in case of missing `networking -> interfaces`
       # section
       NetworkAutoconfiguration.instance.configure_virtuals
-      NetworkAutoconfiguration.instance.configure_dns unless Mode.autoinst
+
+      if !Mode.autoinst
+        NetworkAutoconfiguration.instance.configure_dns
+        configure_network_manager
+      end
 
       # this depends on DNS configuration
       configure_hosts
     end
 
+    # Configures NetworkManager
+    #
+    # When running the live installation, it is just a matter of copying
+    # system-connections to the installed system. In a regular installation,
+    # write the settings in the Yast::Lan.yast_config object.
+    def configure_network_manager
+      return unless Y2Network::ProposalSettings.instance.network_service == :network_manager
+
+      if Yast::Mode.live_installation
```

Comment:
This mode is still valid? I think that we were not able to rely on it anymore and it is detected as normal mode or something like that.

Reply:
We probably should document it in the code:

See https://trello.com/c/ueqrCN8I/3630-improve-live-installation-usability-and-behavior

Reply:
Good point. I have updated the code to rely on `Yast::Lan.system_config.backend` as you suggested offline.